pkg/gui/refresh.go

```diff
@@ -554,6 +571,10 @@ func (gui *Gui) refreshStatus() {
 
 	name := presentation.GetBranchTextStyle(currentBranch.Name).Sprint(currentBranch.Name)
 	repoName := utils.GetCurrentRepoName()
+	mainWorktreeName := gui.helpers.Worktree.GetMainWorktreeName()
+	if repoName != mainWorktreeName {
+		repoName = fmt.Sprintf("%s(%s)", mainWorktreeName, style.FgBlue.Sprint(repoName))
```

Comment:
I'd use something other than blue given it can be hard to see on some screens (we typically use blue for unimportant stuff but I think it's important to show what worktree the user is in). How bout magenta?